src/radha/Assignment8.java

```diff
@@ -0,0 +1,53 @@
+package radha;
+
+public class Assignment8 {
+	void evenNumbers(int startIndex, int endIndex) {
+		int i = startIndex;
```

Comment:
This extra variable not require, you can handle this without using that variable,
Kindly refer below code for the same.

https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files